engine/src/out

```diff
@@ -0,0 +1 @@
+/media/oldhome/engine/out
```

Comment:
Looks like an unintentional change

Reply:
Exactly, this a change specific to my engine setup that should not be commited.